src/Raven.Server/Documents/ETL/Providers/Raven/RavenEtlDocumentTransformer.cs

```diff
@@ -344,6 +405,159 @@ private void AddSingleCounterGroup(LazyStringValue docId, BlittableJsonReaderObj
                     _currentRun.AddCounter(docId, prop.Name, value);
             }
         }
+        
+        private void AddTimeSeries(LazyStringValue docId, Dictionary<string, IEnumerable<TimeSeriesStorage.Reader.SingleResult>> timeSeriesReaders)
```

Comment:
Right now we are sending the full segment when it changes, one entry at a time.
Need to consider how we can send a whole segment as a unit.
Because otherwise if we have 10 adds, we'll send batches of: 1,2,3,4,5,6,7,8,9
Total of 45.

Reply:
We can't know at any point if there will be more appends, right?
So we can't wait for segment to be full.
Maybe a delay by time?
And we can let change the value by configuration or somehow defined it in the script